tests/core/test_dataset_filters.py

```diff
@@ -1030,12 +1030,14 @@ def test_glyph_orient_and_scale():
 
 @pytest.mark.parametrize('color_mode', ['scale', 'scalar', 'vector'])
 def test_glyph_color_mode(sphere, color_mode):
-    assert sphere.glyph(color_mode=color_mode)
+    # define vector data
+    sphere.point_data['velocity'] = sphere.points[:, [1, 0, 2]] * [-1, 1, 0]
+    sphere.glyph(color_mode=color_mode)
```

Comment:
This was the source of 2 remaining fixable warnings. Two remarks:
1. I removed the `assert` because even empty `PolyData` are truthy
2. the real test of this feature would be to do an image regression test... I don't know if we could squeeze it in somewhere in `tests/plotting`. Would that make sense? Should I open an issue for it?

Reply:
This just tests calling the vtk method.  While a test is nice to be in the very safe side, I don't think it is necessary.  Definitely not for this PR.

Reply:
Thanks @MatthewFlamm, I won't open an issue then. (I had figured we could test that we're dispatching to the correct VTK method.)